livekit-android-sdk/src/main/java/io/livekit/android/room/track/TrackPublication.kt

```diff
@@ -22,10 +22,6 @@ open class TrackPublication(
         private set
     open var muted: Boolean = false
         internal set
-    open val subscribed: Boolean
```

Comment:
this is kind of useful when you have a TrackPublication, and don't care if it's remote or local, but just need to quickly check isSubscribed then decide to render.